src/interpolators.jl

```diff
@@ -83,6 +83,11 @@ Really fast interpolation that happens on the GPU (for GLMakie), so optimal for
 
 !!! warning
     `DelaunayMesh` won't allow you to add a contour plot to the topoplot.
+
+!!! danger
+    `DelaunayMesh` will not behave accurately if rendered via CairoMakie, because Cairo (and SVG in general)
```

Comment:
is this true when using CairoMakie with PNG output?

Reply:
Yes, Cairo only receives color information at the vertices of each mesh triangle.